### DISCUSSION TITLE
Error when perform git clone

### DISCUSSION THREAD

**Discussion by marlina107** [community/community#137691] [Programming Help]
### Select Topic Area

Question

### Body

Hi all,
Below is the error when i try to clone github. I already downloaded git cli but still facing an error.
![Screenshot 2024-09-04 160743](https://github.com/user-attachments/assets/3f02342b-446f-4604-be75-6997b85480d3)

**Comment by Aakash-Dutta** [comment]
It seems you have not added git to your path. Follow the instruction in this [link](https://linuxhint.com/add-git-to-path-windows/) to add git to your path.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @marlina107 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Thxmxs** [comment]
The error message you're encountering, "git' is not recognized as an internal or external command, operable program or batch file," typically means that Git is either not installed correctly or its path has not been added to the system's PATH environment variable.

Here’s how you can resolve it:

1. Verify Git Installation:
Ensure Git is installed properly. You can download and install Git from the official website: Git for Windows.
During the installation, make sure to select the option to "Add Git to PATH" when prompted.

2. Check Git Path in Environment Variables:
If Git is installed but not recognized, you may need to manually add Git to your system’s PATH.

Steps:

1.Press Windows + S, type "Environment Variables," and open "Edit the system environment variables."
2.In the window that opens, click the Environment Variables button at the bottom.
4.In the System Variables section, scroll and find the Path variable, then select and click Edit.
5.In the Edit Environment Variable window, click New and add the following path (assuming you installed Git in the default location):
C:\Program Files\Git\bin
C:\Program Files\Git\cmd
Click OK to close all windows
Verify the Installation:
After updating the PATH, open a new Command Prompt or PowerShell window and type:
git --version
If Git is correctly installed, it should output the installed version of Git.

Once Git is recognized, you should be able to use git clone or any other Git commands.
